wordpress-mobile/WordPress-iOS
# Merge UIImageView Gravatar extensions

Merged all the objective-C UIImageView+Gravatar extensions into the Swift file. Removed the objective-C files and the related inclusions. Fixed a couple of calls that went broken.

Fixes #8235 

To test:
1. Launch the app and log-in
2. Verify that all the blog's gravatar are shown in the grid (check with both blogs with a set icon and blogs without a set icon. The default one should be shown in the latest case)
3. Select a blog and verify that the gravatar is shown in the property page.
